extensions/ql-vscode/src/vscode-tests/minimal-workspace/determining-selected-query-test.ts

```diff
@@ -0,0 +1,44 @@
+import { expect } from 'chai';
+import * as path from 'path';
+import * as vscode from 'vscode';
+import { Uri } from 'vscode';
+import { determineSelectedQuery } from '../../run-queries';
+
+async function showQlDocument(name: string): Promise<vscode.TextDocument> {
+  const folderPath = vscode.workspace.workspaceFolders![0].uri.fsPath;
+  const documentPath = path.resolve(folderPath, name);
+  const document = await vscode.workspace.openTextDocument(documentPath);
+  await vscode.window.showTextDocument(document!);
+  return document;
+}
+
+export function run() {
+  describe('Determining selected query', async () => {
+    it('should allow ql files to be queried', async () => {
+      const q = await determineSelectedQuery(Uri.parse('file:///tmp/queryname.ql'), false);
+      expect(q.queryPath).to.equal('/tmp/queryname.ql');
```

Comment:
This fails on windows. Use `path.separator` instead of `/`.

Reply:
Oof, thanks for noticing.